utils/node_local_test.py

```diff
@@ -3173,6 +3178,59 @@ def test_daos_fs_check(self):
         if dir_nr != 11:
             raise NLTestFail('Wrong number of sub-directories in lost+found')
 
+        # punch the test_dir2 object.
+        # this makes test_dir2 an empty dir (leaking everything under it)
+        cmd = [daos_mw_fi, pool, cont, "punch_obj", "/test_dir2"]
+        print(cmd)
+        rc = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                            env=cmd_env, check=False, cwd=self.dfuse.dir)
+        if rc.stderr != b'':
+            print('Stderr from command')
+            print(rc.stderr.decode('utf-8').strip())
+        if rc.stdout != b'':
+            print(rc.stdout.decode('utf-8').strip())
+        assert rc.returncode == 0
```

Comment:
More a question for @ashleypittman - Instead of assert, should this `raise NLTestFail`?

Reply:
fwiw i just copied this from somewhere in nlt

Reply:
I'm a fan of using assert for checking in tests, sometimes it's easier to raise an exception but to me that's really only a requirement if you want to pass information out so the harness can report/skip/retry etc accordingly.